servers/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_servers"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic"
```

Comment:
Change this repository to "https://github.com/EpicCash/epic"

Reply:
Done